### PR TITLE
Plus operator overloading

### PR DESCRIPTION
**Pull request by hakhov**

### CODE REVIEW

File: Codes/Student.cs [hunk 1]
```diff
@@ -14,7 +14,12 @@ public List<Student> Students()
...
 
-		// Write your code here to compile code and pass test
-
+		public static List<Student> operator +(Student obj1, Student obj2)
+		{
+			var students = new List<Student>();
+			students.Add(obj1);
+			students.Add(obj2);
+			return students;
+		}
```
Comment (GalstyanTsolak) [inline Codes/Student.cs:23]: **The right approach, but not the best practice implementation. Think better**

Comment (hakhov) [reply Codes/Student.cs]: I did some improvementt

File: Codes/Student.cs [hunk 2]
```diff
@@ -14,7 +14,6 @@ public List<Student> Students()
 			return john + bob;
 		}
 
-		// Write your code here to compile code and pass test
-
+		public static List<Student> operator +(Student obj1, Student obj2) => new List<Student> { obj1, obj2 };
```
Comment (GalstyanTsolak) [inline Codes/Student.cs:17]: `obj1` and `obj2` is not right naming, In my opinion `(Student a, Student b)` more readable.

**Review by GalstyanTsolak — CHANGES_REQUESTED** [review]

File: Codes/Student.cs (hunk 2 above)
Comment (hakhov) [reply Codes/Student.cs]: I avoid talking about single-letter variables, I changed them.

**Pull request merged**